wgpu-hal/src/gles/mod.rs

```diff
@@ -188,6 +188,7 @@ struct AdapterShared {
 
 pub struct Adapter {
     shared: Arc<AdapterShared>,
+    downlevel_flags: wgt::DownlevelFlags,
```

Comment:
Because of my other comment, I feel like we should store "is_ext_color_buffer_float_supported" as a boolean, and it should live inside the shared adapter.

Reply:
Done